### DISCUSSION TITLE
enum_def supports with custom table name

### DISCUSSION THREAD

**Discussion by dale-vendia** [SeaQL/sea-query#768] [Q&A] [answered]
Does anyone know if the enum_def supports a custom table name? 

It appears to be by convention (the name of the struct), and by reading the macro, I'm unable to tell if you can customise it. https://github.com/SeaQL/sea-query/tree/master/sea-query-attr

My current workaround is to use an enum JUST for the name and an enum_def for the properties. Though, this feels clunky.

```
#[derive(Copy, Clone, Iden, EnumIter)]
enum Transaction {
    #[iden = "transaction"]
    Table,
}
```

**Comment by tyt2y3** [comment, marked as answer]
It does, but only since two days ago https://github.com/SeaQL/sea-query/pull/759
